inst/report/Report_KRI.Rmd

```diff
@@ -143,11 +140,10 @@ for (i in unique(params$dfMetrics$MetricID)) {
 ```{r, echo=FALSE, results='asis'}
 #print dfMetrics table
 params$dfMetrics %>%
-kbl(format="html", escape=FALSE) %>%
-kable_styling ("striped", full_width = FALSE) %>%
-cat
+  gsm_gt(id = "metrics_table")
```

Comment:
This function (`gsm_gt()`) isn't exported, so this fails if you install the package and run the code as an actual user:
```r
lCharts <- MakeCharts(
  dfResults = gsm::reportingResults,
  dfGroups = gsm::reportingGroups,
  dfMetrics = gsm::reportingMetrics,
  dfBounds = gsm::reportingBounds
)

Report_KRI(
  lCharts = lCharts,
  dfResults =  gsm::reportingResults,
  dfGroups =  gsm::reportingGroups,
  dfMetrics = gsm::reportingMetrics,
  strOutputDir = here::here("pkgdown", "assets"),
  strOutputFile = "report_kri_site.html"
)
```
```
! could not find function "gsm_gt"
```

Either export `gsm_gt()` (but that's just meant as a convenience wrapper to apply any styles we decide to implement), or use `gt::gt()` directly.